.github/workflows/turbo-affected-reusable.yml

```diff
@@ -37,6 +37,8 @@ jobs:
     steps:
       - name: Checkout head branch
         uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.head_branch }}
```

Comment:
*excludes* the merge commit

therefore turbo affected will compare the head of this branch to the base branch